crates/uv-cli/src/lib.rs

```diff
@@ -4672,6 +4672,19 @@ pub struct PythonPinArgs {
     /// `requires-python` constraint.
     #[arg(long, alias = "no-workspace")]
     pub no_project: bool,
+
+    /// Pin to a specific Python version.
```

Comment:
This still needs to explain the behavior of this flag.

Just copying the previous suggestion:

```suggestion
    /// Update the global Python version pin.
```